src/components/Calculator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => {
+  const getButtonStyle = (value) => {
+    const buttonStyles = {
+      0: 'zero',
+      x: 'operator',
+      '-': 'operator',
+      '+': 'operator',
+      '=': 'operator',
+      '÷': 'operator',
+    };
+    return `button ${buttonStyles[value]}`;
+  };
+
+  return (
```

Comment:
- [ OPTIONAL ] Great job working on the requested changes but I think you missed one important requirement to Create at least 1 child component for the calculator component so kindly create a new component and include it to the calculator 🚀🤩 I assign  it as optional because the previous reviewer forget to mentioned that but it still important to add🤩🚀

![image](https://github.com/mryeminthu/math-magicians/assets/58553711/77a0d7ce-60b5-467c-88ce-2970179784f5)